elements/pfe-clipboard/src/pfe-clipboard.html

```diff
@@ -0,0 +1,14 @@
+<div class="pfe-clipboard__container">
```

Comment:
You could add the `role="button"` here directly or even just use a button tag in the template.

Reply:
Removed the container, much better!  I added the tabindex and role to properties object. Also hid those properties on the storybook.